### ISSUE TITLE
Improve pipelines

### ISSUE THREAD

**Issue by Grokzen** [Grokzen/redis-py-cluster#57]
There is alot of work that needs to be done to make pipelines work at the same level as the normal client.

- [ ] Work to unblock the basic commands that is currently blocked (mget, mset etc...)
- [ ] Take another look at NYI commands like `immediate_execute_command` & `_execute_transaction` to see if they can be implemented.
- [ ] Split `send_cluster_commands` into smaller parts that can be reused more easily.

**Comment by Grokzen** [comment]
Closing this in favor for other/newer issues.
